ui/src/components/DataTypeField.vue

```diff
@@ -59,13 +51,25 @@ import DataTypeParamField from './DataTypeParamField.vue'
 export default class extends Vue {
   @Prop() value: DataType | null
   @Prop() label: string
-  matchingDatatypes = datatypes.all
   showParams = false
 
-  typeName: string = datatypes.byURI(this.value?.id ?? '')?.name ?? ''
+  get datatypes () {
+    return datatypes.all.concat().sort((dt1, dt2) => {
+      const name1 = dt1.name.toUpperCase()
+      const name2 = dt2.name.toUpperCase()
+
+      if (name1 < name2) {
+        return -1
+      } else if (name1 > name2) {
+        return 1
+      } else {
+        return 0
+      }
```

Comment:
Would this be equivalent?

```
return datatypes.all.concat().sort((dt1, dt2) => dt1.localeCompare(dt2))
```

Reply:
Let me check, that would be awesome :)

Reply:
It works! I learned something :) (TBH I was very surprised to have to write the comparison function by hand... guess I didn't have to)